### ISSUE TITLE
Bruno installed via snap on Xubuntu is crashing

### ISSUE THREAD

**Issue by FrontEndStudio** [usebruno/bruno#655]
# Problem

Bruno installed via snap on Xubuntu is crashing on startup.

# Bruno version:
```
snap list  | grep bruno 
bruno                      0.25.0                      3      latest/stable  helloanoop        -
```

# Gtk version:
```
gtk-launch --version
3.24.33
```

 # Xubuntu version:
```
lsb_release -a 
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy
```

Error message:
```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '/snap/bruno/3/resources/app.asar/node_modules/@usebruno/query/dist/cjs/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:360:19)
    at Module._findPath (node:internal/modules/cjs/loader:573:18)
    at Module._resolveFilename (node:internal/modules/cjs/loader:926:27)
    at n._resolveFilename (node:electron/js2c/browser_init:249:1105)
    at Module._load (node:internal/modules/cjs/loader:785:27)
    at c._load (node:electron/js2c/asar_bundle:5:13339)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/snap/bruno/3/resources/app.asar/node_modules/@usebruno/js/src/utils.js:2:17)
    at Module._compile (node:internal/modules/cjs/loader:1120:14)
Gtk-Message: 11:29:12.811: Failed to load module "xapp-gtk3-module"

(bruno:24205): Gtk-WARNING **: 11:29:12.857: Theme parsing error: gtk.css:1374:85: 'font-feature-settings' is not a valid property name
```

**Comment by FrontEndStudio** [comment]
BTW your .deb package works perfect, thank you for building this great product.

```
dpkg -i bruno_0.25.0_amd64_linux.deb 
Selecting previously unselected package bruno.
(Reading database ... 309523 files and directories currently installed.)
Preparing to unpack bruno_0.25.0_amd64_linux.deb ...
Unpacking bruno (0.25.0) ...
Setting up bruno (0.25.0) ...
```

**Comment by helloanoop** [comment]
@FrontEndStudio 

Glad that you like Bruno!

The snap build has been fixed in the latest release `v0.26.0` . Can you try again ?

**Comment by helloanoop** [comment]
Closing this issue, please re-open if snap builds still continue to fail.
